modules/database/audit-autoloaded-options/hooks.php

```diff
@@ -20,9 +20,119 @@
  */
 function perflab_aao_add_autoloaded_options_test( $tests ) {
 	$tests['direct']['autoloaded_options'] = array(
-		'label' => __( 'Autoloaded options', 'performance-lab' ),
+		'label' => esc_html__( 'Autoloaded options', 'performance-lab' ),
```

Comment:
I don't see where this label is ever shown. It may not need `esc_html()`.

Reply:
Yes, this doesn't need to be escaped, it's handled by core. Also somewhat unrelated to this PR anyway.